feat(replay): Add docs for scrubbing URLs on the replay event (#9058)
getsentry/sentry-docs: docs/platforms/javascript/common/session-replay/privacy.mdx

We also have [server-side PII scrubbing](/product/data-management-settings/scrubbing/server-side-scrubbing/) for this data. It looks for certain patterns such as American social security numbers, credit cards, and private keys.

### Example: Scrubbing URLs

> URLs can be stored in the recording event (which you can use the above `beforeAddRecordingEvent` to scrub), but they can also be stored in a replay event as well. In order to scrub the latter case, you must use `addEventProcessor`:

Reviewer: ```suggestion
By default, URLs are stored in both the recording event and the replay event.

To scrub the URL in the recording event, you can use the above `beforeAddRecordingEvent`. 

To scrub the URL in replay events, you must use `addEventProcessor`:
```

We also have [server-side PII scrubbing](/product/data-management-settings/scrubbing/server-side-scrubbing/) for this data. It looks for certain patterns such as American social security numbers, credit cards, and private keys.

### Example: Scrubbing URLs

> 
> By default, URLs are stored in both the recording event and the replay event.
> 
> To scrub the URL in the recording event, you can use the above `beforeAddRecordingEvent`. 
> 
> To scrub the URL in replay events, you must use `addEventProcessor`:

Reviewer 2: ```suggestion

By default, URLs are stored in both recording and replay events.

To scrub the URL in a recording event, use the above `beforeAddRecordingEvent`. 

To scrub the URL in a replay event, use `addEventProcessor`:
```
